### ISSUE TITLE
Option to not have any chat selected

### ISSUE THREAD

**Issue by kuba-orlik** [vector-im/element-web#25847]
#### What would you like to do?

I would like to have the option to close the current chat and not have any chat selected at a given time. This could be achieved by pressing Esc, for example.

#### Why would you like to do it?

To avoid this scenario:

1. I'm chatting with someone so their chat is currently active
2. I'm ctrl+tabbing to a different tab
3. the user I'm chatting with replies after a few hours, so the chat is marked as having unread messages
4. I'm ctrl+tabbing across my tabs in the browser, so the tab with element is active for a moment
5. element treats it as me having read the message, but I didn't even see it
6. I miss the message because later when I come back it's not marked as unread

#### How would you like to achieve it?

Pressing Esc could just close the current chat and display an empty space in the chat area. Telegram is doing that, for example, and I LOVE this feature.

### Have you considered any alternatives?

One alternative could be marking the messages as read only after some significant mouse movement or some keyboard input, not just activating the tab

**Comment by t3chguy** [comment]
Duplicate of https://github.com/vector-im/element-web/issues/23369
